internal/assistedserviceiso/assistedserviceiso_api.go

```diff
@@ -90,7 +90,7 @@ func (a *assistedServiceISOApi) CreateISOAndUploadToS3(ctx context.Context, para
 	ignitionConfig := reIgnition.Replace(ignitionConfigSource)
 
 	username := ocm.UserNameFromContext(ctx)
-	srcISOName, err := a.objectHandler.GetBaseIsoObject(params.AssistedServiceIsoCreateParams.OpenshiftVersion)
+	srcISOName, err := a.objectHandler.GetBaseIsoObject(params.AssistedServiceIsoCreateParams.OpenshiftVersion, "x86_64")
```

Comment:
```suggestion
	srcISOName, err := a.objectHandler.GetBaseIsoObject(params.AssistedServiceIsoCreateParams.OpenshiftVersion, common.TestDefaultConfig.CPUArchitecture
```

Reply:
probably not the one from a `TestDefault` maybe the one from versions.

Reply:
right :)